.github/workflows/_build_artifacts.yml

```diff
@@ -171,7 +171,7 @@ jobs:
             platform: linux/arm/v7
             shortname: arm
         name:
-          - package: firezone-linux-client
+          - package: firezone-headless-client
             artifact: linux-client
             image_name: client
```

Comment:
```suggestion
            image_name: headless-client
```

Reply:
I think simpler might be better, by virtue of it being a Docker image I think that `client` suffices